src/objects/npe03__Recurring_Donations_Settings__c.object

```diff
@@ -1,5 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fields>
+        <fullName>DataMigrationBatchSize__c</fullName>
+        <defaultValue>200</defaultValue>
+        <description>The number of records to process at a time when migrating existing Recurring Donation records to Enhanced Recurring Donations format.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The number of records to process at a time when migrating existing Recurring Donation records to Enhanced Recurring Donations format.</inlineHelpText>
```

Comment:
@sam-knox Would you mind doing a quick review of this new field label and help text? This field will not be visible in the NPSP Settings UI though.